src/routes/docs/tutorials/android/step-6/+page.markdoc

```diff
@@ -0,0 +1,103 @@
+---
+layout: tutorial
+title: Add database
+description: Add databases and queries to store user data in you Android application.
+step: 6
+---
+# Create collection
+In Appwrite, data is stored as a collection of documents. Create a collection in the [Appwrite Console](https://cloud.appwrite.io/) to store our ideas.
+
+{% only_dark %}
+![Create project screen](/images/docs/tutorials/dark/idea-tracker-collection.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/tutorials/idea-tracker-collection.png)
+{% /only_light %}
+
+Create a new collection with the following attributes:
+| Field       | Type   | Required |
+|-------------|--------|----------|
+| userId      | String | Yes      |
```

Comment:
Would it be better to also set permissions so users would only see their own ideas?